tests/unit_tests/test_wave_06.py

```diff
@@ -110,7 +113,17 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c in jesse.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
```

Comment:
How could this been done with a loop? If done with a loop, what the time complexity be? Based off of the time complexity should we implement a loop?

Reply:
`items = [item_f, item_a, item_b]
for item in items:
    assert item in tai.inventory == True
`
Never implemented this so not 100% sure this will work.  Lemme know ok?
The time complexity would be the same because both solutions are O(n)?  Though in the solution just implemented, would space complexity be O(n) because we are making a new data structure to hold the items?  Whereas in the first solution we are directly retrieving the items from memory?